src/lib/converter/plugins/GitHubPlugin.ts

```diff
@@ -1,20 +1,21 @@
-import * as ShellJS from "shelljs";
-import * as Path from "path";
+import * as ShellJS from 'shelljs';
+import * as Path from 'path';
 
-import {ISourceReference} from "../../models/sources/file";
-import {Component, ConverterComponent} from "../components";
-import {BasePath} from "../utils/base-path";
-import {Converter} from "../converter";
-import {Context} from "../context";
+import {ISourceReference} from '../../models/sources/file';
+import {Component, ConverterComponent} from '../components';
+import {BasePath} from '../utils/base-path';
+import {Converter} from '../converter';
+import {Context} from '../context';
 
 // This should be removed when @typings/shelljs typings are updated to the shelljs version being used
-declare module "shelljs" {
+declare module 'shelljs' {
     // `stdout` was added in:
     // https://github.com/shelljs/shelljs/commit/8a7f7ceec4d3a77a9309d935755675ac368b1eda#diff-c3bfabb5e6987aa21bc75ffd95a162d6
     // As of 2016-10-16, DefinitelyTyped's defs are for shelljs v0.3.0, but we're on 0.7.0
+    // tslint:disable-next-line:interface-name
```

Comment:
Possibly my fault when I started updating TypeDoc and needed to improve that definition. I'd just check the blame, but it can be made consistent. I'd love to have another pass through removing the prefixes and correcting whitespace and whatnot.